oic.r.cred.raml

```diff
@@ -95,8 +118,19 @@ traits:
         description: The request is invalid.
 
   post:
+
```

Comment:
The discussion that I recall is that we would retain the dual "rowner" and "rowneruuid" elements in ACL2 even though rowner can express host ownership in the form of UUIDs - and that "rowneruuid" could/should be deprecated across all SVRs as part of a since CR.